### PR TITLE
docs: add precisions in readme

### PR DESCRIPTION
**Pull request by jacques-lebourgeois**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,47 @@
...
+  }
+);
+
+// Enriched dataOptions to complete implementation of the Orange Design System.
+myChart.setOption(themeManager.getChartOptions(dataOptions));  
+
+```
+The themeManager returned by getThemeManager() can be used to add other features supported by the ODS Charts library. Please refer to [the online documentation for more details](https://ods-charts.netlify.app/). 
+
+## Quick start for contributors
```
Comment (julien-deramond) [inline README.md:63]: I think this part doesn't really work with what's inside

> - Install with [npm](https://www.npmjs.com/): `npm install ods-charts`
> Read the [documentation](https://ods-charts.netlify.app/) for information on the library content and examples.

It's for the users too.

We could keep just a "Quick start" and mix both contents

File: README.md [hunk 2]
```diff
@@ -20,7 +20,47 @@
...
+  {
+    renderer: 'svg', // we still prefer svg renderer
+  }
+);
+
+// Enriched dataOptions to complete implementation of the Orange Design System.
+myChart.setOption(themeManager.getChartOptions(dataOptions));  
+
+```
+The themeManager returned by getThemeManager() can be used to add other features supported by the ODS Charts library. Please refer to [the online documentation for more details](https://ods-charts.netlify.app/).
```
Comment (julien-deramond) [inline README.md:61]:
```suggestion
The `themeManager` returned by `getThemeManager()` can be used to add other features supported by the ODS Charts library. Please refer to [the online documentation for more details](https://ods-charts.netlify.app/). 
```

File: README.md [hunk 3]
```diff
@@ -20,7 +20,47 @@
...
+```
+var div = document.getElementById('barLine_chart');
+var myChart = echarts.init(div, undefined, {
+  renderer: 'svg', // we prefer svg renderer
+});
+// Display the chart.
+myChart.setOption(dataOptions);
+```
+
+Adding the theme managed by ODS Charts will then consist of doing :
```
Comment (julien-deramond) [inline README.md:39]:
```suggestion
Adding the theme managed by ODS Charts will then consist of doing:
```

File: README.md [hunk 4]
```diff
@@ -20,7 +20,47 @@
...
+```
```
Comment (julien-deramond) [inline README.md:40]: You can precise the language here for syntax highlighting

File: README.md [hunk 5]
```diff
@@ -20,7 +20,47 @@
...
+});
+
+// register this theme to echarts
+echarts.registerTheme(themeManager.name , themeManager.theme);
+
+var div = document.getElementById('barLine_chart');
+var myChart = echarts.init(div, 
+  themeManager.name,  // initialized the chart with the generated theme 
+  {
+    renderer: 'svg', // we still prefer svg renderer
```
Comment (julien-deramond) [inline README.md:53]:
```suggestion
    renderer: 'svg', // SVG rendered is recommended
```

File: README.md [hunk 6]
```diff
@@ -20,7 +20,47 @@
...
+var themeManager = ODSCharts.getThemeManager({
+  // you can configure here some optional options
+});
+
+// register this theme to echarts
+echarts.registerTheme(themeManager.name , themeManager.theme);
+
+var div = document.getElementById('barLine_chart');
+var myChart = echarts.init(div, 
+  themeManager.name,  // initialized the chart with the generated theme
```
Comment (julien-deramond) [inline README.md:51]:
```suggestion
  themeManager.name,  // initialize the chart with the generated theme 
```

File: README.md [hunk 7]
```diff
@@ -20,7 +20,47 @@
...
+var myChart = echarts.init(div, undefined, {
+  renderer: 'svg', // we prefer svg renderer
+});
+// Display the chart.
+myChart.setOption(dataOptions);
+```
+
+Adding the theme managed by ODS Charts will then consist of doing :
+```
+// Build the theme
```
Comment (julien-deramond) [inline README.md:41]:
```suggestion
// build the theme
```

to be consistent with the other comments

File: README.md [hunk 8]
```diff
@@ -20,7 +20,47 @@
...
+## Quick start for users
+The ODS Chart library is to be used with Apache ECharts to produce charts in line with the Orange Design System.
+
+It has been designed to be minimally intrusive and not to interfere with the use of the Apchae Echarts library. The Apache Echarts documentation remains the reference for configuring graphics.
+
+For example, an Apache Echarts user will initialize a graph with code like this:
+```
+var div = document.getElementById('barLine_chart');
+var myChart = echarts.init(div, undefined, {
+  renderer: 'svg', // we prefer svg renderer
```
Comment (julien-deramond) [inline README.md:33]:
```suggestion
  renderer: 'svg', // SVG is recommended
```

File: README.md [hunk 9]
```diff
@@ -20,7 +20,47 @@
...
 
-## Quick start
+
+## Quick start for users
+The ODS Chart library is to be used with Apache ECharts to produce charts in line with the Orange Design System.
+
+It has been designed to be minimally intrusive and not to interfere with the use of the Apchae Echarts library. The Apache Echarts documentation remains the reference for configuring graphics.
+
+For example, an Apache Echarts user will initialize a graph with code like this:
+```
```
Comment (julien-deramond) [inline README.md:30]: Precise the language for syntax highlighting

File: README.md [hunk 10]
```diff
@@ -20,7 +20,47 @@
 - [Versioning](#versioning)
 - [Copyright and license](#copyright-and-license)
 
-## Quick start
+
+## Quick start for users
+The ODS Chart library is to be used with Apache ECharts to produce charts in line with the Orange Design System.
```
Comment (julien-deramond) [inline README.md:25]:
```suggestion
The ODS Charts library is to be used with Apache ECharts to create charts that adhere to the Orange Design System guidelines..
```

File: README.md [hunk 11]
```diff
@@ -20,7 +20,47 @@
...
+
+It has been designed to be minimally intrusive and not to interfere with the use of the Apchae Echarts library. The Apache Echarts documentation remains the reference for configuring graphics.
```
Comment (julien-deramond) [inline README.md:27]:
```suggestion
It has been designed to be minimally intrusive and not to interfere with the use of the Apache ECharts library. The Apache ECharts documentation remains the reference for configuring charts.
```

File: README.md [hunk 12]
```diff
@@ -20,7 +20,47 @@
 - [Versioning](#versioning)
 - [Copyright and license](#copyright-and-license)
 
-## Quick start
+
+## Quick start for users
```
Comment (julien-deramond) [inline README.md:24]: Here, we could keep "Quick start"

**Review by julien-deramond — CHANGES_REQUESTED** [review]
I haven't checked everything, but here are a few comments already.
Plus, Prettier job is failing.

File: README.md (hunk 1 above)
Comment (jacques-lebourgeois) [reply README.md:63]: Agree. I think it is the clone repo that was useless. I 'll made a proposition

Comment (julien-deramond) [reply README.md]: Yes, we can drop the `git clone` part as it's only useful for contributors, and is already mentioned here: https://github.com/Orange-OpenSource/ods-charts/blob/main/CONTRIBUTING.md#pull-requests

File: README.md [hunk 13]
```diff
@@ -24,8 +24,55 @@
 
 Several quick start options are available:
 
-- Clone the repo: `git clone https://github.com/Orange-OpenSource/ods-charts.git`
-- Install with [npm](https://www.npmjs.com/): `npm install ods-charts`
+- npm installation:
+  - Install with [npm](https://www.npmjs.com/): `npm install ods-charts`
+  - Use in your js files : `import * as ODSCharts from 'ods-charts';`
```
Comment (julien-deramond) [inline README.md:29]:
```suggestion
  - Use in your JavaScript files: `import * as ODSCharts from 'ods-charts';`
```

File: README.md [hunk 14]
```diff
@@ -24,8 +24,55 @@
...
+- Add javascript tag in your HTML page:
```
Comment (julien-deramond) [inline README.md:30]:
```suggestion
- Add JavaScript script in your HTML page:
```

File: README.md [hunk 15]
```diff
@@ -24,8 +24,55 @@
...
+
+```html
+<script src=" https://cdn.jsdelivr.net/npm/echarts@latest/dist/echarts.min.js "></script>
```
Comment (julien-deramond) [inline README.md:33]: Are we sure we want to recommend this approach in terms of eco-design? Users should probably only load what they need, and not the entire Apache ECharts library.

File: README.md [hunk 16]
```diff
@@ -24,8 +24,55 @@
 
 Several quick start options are available:
```
Comment (julien-deramond) [inline README.md:25]: So there's only one option now, isn't it?

**Review by julien-deramond — CHANGES_REQUESTED** [review]

File: README.md (hunk 15 above)
Comment (louismaximepiton) [reply README.md:33]: Maybe not recommendation but it's still a way to achieve it

File: README.md (hunk 13 above)
Comment (louismaximepiton) [reply README.md:29]: Maybe here we could redirect to https://apache.github.io/echarts-handbook/en/basics/import/

File: README.md (hunk 15 above)
Comment (jacques-lebourgeois) [reply README.md:33]: I can remove reference to cdn.jsdeliver.net and replace it by a [path to the echarts.min.js]

and keep the npm install for both methods

Comment (jacques-lebourgeois) [reply README.md:33]: [jsDelivr not recomended](https://github.com/Orange-OpenSource/ods-charts/pull/275/commits/4271b0aa85ed5f75f9aaff35d3c75d4e40ea10b4)

Comment (louismaximepiton) [reply README.md:33]: Yes I agree with you (even more since we are still using it in the doc :eyes:)

**Review by louismaximepiton — APPROVED** [review]
👌

File: README.md (hunk 15 above)
Comment (julien-deramond) [reply README.md]: We changed a little bit the wording with @louismaximepiton to explain it

File: README.md (hunk 16 above)
Comment (julien-deramond) [reply README.md]: We reformulated it with @louismaximepiton

**Review by julien-deramond — APPROVED** [review]

**Pull request merged**